docs/content/docs/guide/running.md

```diff
@@ -146,7 +146,7 @@ roses are red, violets are blue. pipeline are bound to flake by design.
 You can cancel a running PipelineRun by commenting on the PullRequest.
 
 For example if you want to cancel all your PipelinerRuns you can add a comment starting
-with `/cancel` and all PipelineRun attached to that pull or merge request will be cancelled.
+with `/cancel` and all PipelineRun attached to that push, pull or merge request will be cancelled.
```

Comment:
can you add a section explaining how to issue a gitops comment on push in github and mentions it only works on github. no need to duplicate the blocks, just a simple example.

Reply:
any chances to have this ?

Reply:
Yes making it ready today EOD

Reply:
it's a public holiday for you so this can wait tomo :)